README.md

```diff
@@ -1,31 +1,30 @@
 # Cloud Code for Visual Studio Code
 
-Cloud Code for VS Code extends VS Code to bring all the power and convenience
-of IDEs to developing cloud-native Kubernetes applications. With Google’s
-command-line [container tools][1] like [`skaffold`][2] and [`kubectl`][3]
-under the hood, Cloud Code gives you local, continuous feedback on your project
-as you build it, extending this local edit-compile-debug loop to create
-cloud-native Kubernetes environments, on your workstation or in the cloud.
-Support for [deployment profiles][4] lets you define different environments,
-like local development, shared development, test, or production, so you can
-easily test and debug on your workstation or in the cloud.
+Cloud Code for VS Code brings the power and convenience
+of IDEs to developing cloud-native Kubernetes and Cloud Run applications.
+Cloud Code works with Google’s command-line [container tools][1] like [`skaffold`][2]
+and [`kubectl`][3] under the hood, providing local, continuous feedback on your project
+as you build, edit, and run your applications on your workstation or in the cloud.
+Additionally, its support for [deployment profiles][4] lets you define different
+environments, like local development, shared development, test, and production, to
+easily test and debug your app with, locally or in the cloud.
```

Comment:
I think we can remove the deployment profiles part. We moved away from those notion.

Reply:
ah, yes, done!